src/github.com/getlantern/lantern-ui/app/index.html

```diff
@@ -91,10 +91,10 @@
     </div>
     <div id="feedLink" ng-class="{subtle: showNews}">
       <div>
-        <a ng-click="showNewsfeed()" ng-show="!showNews">
+        <a ng-click="showNewsfeed()" ng-show="!showNews && !showError">
           {{ 'SHOW_NEWSFEED' | translate }}
         </a>
-        <a ng-click="hideNewsfeed()" ng-show="showNews">
+        <a ng-click="hideNewsfeed()" ng-show="showNews || showError">
```

Comment:
Ok, apart from what we talked in Slack, this seems to address particularly the case of the issue.